bring-it-up/bring-it-up
# [FE#12] Update Navbar Component

## Description
#### Summary:
- Update Navbar component to match medfi styling

#### Changes:
- Update Navigation.tsx to use material ui and medfi styling
- Import Noto Sans font in index.css
- Update package.json with material ui and react router dom dependencies

#### How to test this PR:
1. Run npm start within client folder
2. Navigation bar appears at the top of the screen

## Screenshots:
<img width="1440" alt="Screen Shot 2022-08-31 at 7 04 47 PM" src="https://user-images.githubusercontent.com/98232527/187816724-6efca78c-3d28-4611-81e4-ce979e691ecf.png">

## Checklist
